src/components/index/Header.js

```diff
@@ -91,8 +91,7 @@ const Header = () => (
                     </h2>
 
                     <a className="btn" href="https://github.com/eclipse-theia/theia" target="_blank" rel="noopener noreferrer">View on GitHub</a>
-                    <a className="btn btn--cta" href="https://gitpod.io/#https://github.com/eclipse-theia/theia" target="_blank" rel="noopener">Try in Gitpod &nbsp;&nbsp;&rarr;</a>
-                    <a className="btn btn--cta" href="/docs/blueprint_download/" rel="noopener">Try on Desktop &nbsp;&nbsp;&rarr;</a>
+                    <a className="btn btn--cta" href="/docs/blueprint_download/" rel="noopener">Try now &nbsp;&nbsp;&rarr;</a>
```

Comment:
What is the reason to rename to "try now"?  "try on desktop" indicated that there are other options (i.e. in the browser) so I would prefer it.

Reply:
I asked @jfaltermeier to change exactly that :). I think it looks strange with the text "Try on Desktop" if there is no other option, but I do not have my heart set on it.
See preview: https://deploy-preview-151--romantic-mcclintock-8d974f.netlify.app/

Reply:
I see :-)

Due to https://github.com/eclipse-theia/theia-website/issues/148#issuecomment-823373640 I would be in favor of "try on desktop", but no strong opinion

Reply:
I am in favor of changing it back whenever we have a second button again and to leave it as is for now.